Pear/Profile/ProfileSummaryTableViewCell.swift

```diff
@@ -69,7 +69,9 @@ class ProfileSummaryTableViewCell: UITableViewCell {
     func configure(for user: User) {
         nameLabel.text = "\(user.firstName) \(user.lastName)"
         netIdLabel.text = "Reach me at \(user.netID)"
-        profileImageView.kf.setImage(with: Base64ImageDataProvider(base64String: user.profilePictureURL, cacheKey: user.netID))
+        if let profilePictureURL = URL(string: user.profilePictureURL ?? "") {
```

Comment:
If you made this a URL in the first place then this would be simpler. If you choose not to, it seems strange to initialize `URL(string: "")`, even if you predict it will return nil. Instead, first unwrap `user.profilePictureURL`, then pass the non optional version into the initializer.